apprentice_teacher.rb

```diff
@@ -1,21 +1,36 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
-
-  def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 800
-    @phase = 3
-  end
+require_relative 'teacher'
+require_relative 'praise'
 
-  def offer_high_five
-    "High five!"
-  end
+class ApprenticeTeacher < Teacher
+  
+  include Praisable
+  # Here i want to only take from the super class these attributes.
+  # But I don't know how to do that
+ 
+  # super( attr_reader :age, :salary, :phase, :target_raise)
+
+  # def inititalize
+  #   super
+  # end  
+    # attr_reader :age, :salary, :phase, :target_raise
+  # attr_accessor :name
+
+  # def initialize(options={})
+  #   @age = options.fetch(:age, 0)
+  #   @name = options.fetch(:name, "")
+  #   @target_raise = 800
+  #   @phase = 3
+  # end
+
+  # def offer_high_five
+  #   "High five!"
+  # end
+  # Praisable.offer_high_five
 
   def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
+    super
+    # @phase = num
+    # "Cool, I've always wanted to teach phase #{num}!"
```

Comment:
If all you're doing is calling super, ditch the method in the subclass entirely.